tests/accounts/test_password_reset.py

```diff
@@ -0,0 +1,27 @@
+from django.test import Client
+from django.urls import reverse
+
+import pytest
+
+from maykin_common.accounts.password_reset import PasswordResetView
+
+pytestmark = [pytest.mark.urls("tests.accounts.urls")]
+
+
+def test_user_cant_access_the_password_reset_view_more_than_5_times(client: Client):
+    url = reverse("admin_password_reset")
+    for _ in range(5):
+        response = client.get(url)
+        assert response.status_code == 200
+
+    response = client.get(url)
+    # 429 Too Many Requests
+    assert response.status_code == 429
+
+
+def test_password_reset_view_attributes():
+    view = PasswordResetView()
+    assert view.throttle_name == "password-reset"
+    assert view.throttle_visits == 5
+    assert view.throttle_period == 60
+    assert view.throttle_methods == ("get",)
```

Comment:
this is just repeating the implementation and does not add any value as a test. it does not test behaviour (`test_user_cant_access_the_password_reset_view_more_than_5_times` does though)